roberto_gongora/tic_tac_toe/TicTacToe.py

```diff
@@ -0,0 +1,138 @@
+import random
+import os
+
+
+class TicTacToe:
+    M = []
```

Comment:
Podrias ver la diferencia entre class and object attribute. Recuerda nombre a tus variable con palabras que se puedan entender **matrix**